### DISCUSSION TITLE
What are the dependencies for using this template for Azure GovCloud deployment?

### DISCUSSION THREAD

**Discussion by AzJosephR** [Azure/Enterprise-Scale#1590] [Q&A]
I may not be looking in the right place to find it, but is there a dependencies list for this template to work in Azure Gov Cloud?

Running from the portal, we've never been able to get any further than what must be a fatal conflict error involving the first alz-Policy being applied.

All the error tells us is:

alz-Policy-usgovvirginia-#GUID# Microsoft.Resources/deployments Conflict

/providers/Microsoft.Resources/deployments/NoMarketplace-20240304181243/operations/D2EAE40F2AAFC82D

{
    "status": "Failed",
    "error": {
        "code": "DeploymentFailed",
        "message": "At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/arm-deployment-operations for usage details.",
        "details": [
            {
                "code": "BadRequest",
                "message": "{\r\n  \"error\": {\r\n    \"code\": \"PolicyDefinitionNotFound\",\r\n    \"message\": \"The policy set definition 'Enforce-EncryptTransit' request is invalid. The following policy definition could not be found: '/providers/Microsoft.Authorization/policyDefinitions/0e80e269-43a4-4ae9-b5bc-178126b8a5cb'.\"\r\n  }\r\n}"
            }
        ]
    }
}

My best guess is there is something which needs to be configured prior to running the template so this policy deployment doesn't fail, but we've had no luck identifying what.

TL;DR
This template will not work in Azure Gov Cloud without one or more dependencies being addressed, and we're not sure how to resolve the issue we're experiencing.

Thanks!
